CHANGELOG.md

```diff
@@ -103,6 +103,9 @@ This document tracks pending changes to packages. It is facilitating the write-u
 
 - Replaced all factories by a single `Factory` for all holding implementations.
 
+- Renamed the `NonTransferable` and `Fungible` implementation to `Base` and
```

Comment:
I would add the new `ensure` clauses to the changelog

Reply:
done